builder/src/scss/theme/_bootswatch.scss

```diff
@@ -254,6 +254,18 @@ $web-font-path: "https://fonts.googleapis.com/css?family=Lato:400,700,400italic"
 
 // Containers ==================================================================
 
+.list-group {
+    border-top-left-radius: 0;
+    border-top-right-radius: 0;
+}
+
+.list-group-item {
+    &:first-child {
+        border-top-left-radius: 0;
+        border-top-right-radius: 0;
+    }
+}
+
```

Comment:
Not a good idea to modify the global theme for changes that we're interested in applying to a specific element. Create a custom class & use that.